### DISCUSSION TITLE
Python "bad interpreter" error when running Fontbakery

### DISCUSSION THREAD

**Discussion by tphinney** [fonttools/fontbakery#3463] [Q&A] [answered]
I had been running Fontbakery often (nearly every day) for the past year or more without problems. After editing my build script (which I do often—it just calls Fontbakery with `fontbakery check-googlefonts fontname.ttf`) and upgrading my macOS to 11.6 with the latest security fixes I got this error trying to run Fontbakery. Perhaps triggered by OS 11.6 upgrade?

> /Library/Frameworks/Python.framework/Versions/3.7/bin/fontbakery: /usr/local/bin/python3: bad interpreter: Operation not permitted

**Comment by tphinney** [comment, marked as answer]
The fix that worked for me was to do this, as recommended by Stack Overflow and elsewhere:

`xattr -d com.apple.quarantine /Library/Frameworks/Python.framework/Versions/3.7/bin/fontbakery`

**Reply by felipesanches** [reply]
thanks!
